kieserver/modules/kieserver/module.yaml

```diff
@@ -36,6 +36,10 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "kie-server-services-jbpm-cluster-7.x.Final.jar"
+  md5: "44f302877d28e2493fc28ead4dfb6dea"
+  # kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00004.jar"
+  dest: /opt
```

Comment:
please, do not just drop files under /opt root path.
create a dir which when someone reads it he knows what the dir is used for.

e.g.
on-demand-deps? or something similar.

Reply:
@spolti  /opt/deps ?

Reply:
/opt/deps